packages/abstract-core/src/native/version_control.rs

```diff
@@ -40,6 +44,15 @@ pub mod state {
     pub const REGISTERED_MODULES: Map<&ModuleInfo, ModuleReference> = Map::new("module_lib");
     // Yanked Modules
     pub const YANKED_MODULES: Map<&ModuleInfo, ModuleReference> = Map::new("yanked_modules");
+    // Modules Fee
+    pub const MODULE_MONETIZATION: Map<(Namespace, String), Monetization> =
+        Map::new("yanked_modules");
```

Comment:
this namespace is wrong.

Reply:
Corrected here : https://github.com/AbstractSDK/contracts/pull/358/files#diff-4e6d021cf259966a3fb05c3c5e0a06618d9c2962705c9e38c425d3885ea85ad2R49